src/components/ScanPatientPage/ScanPatientPage.tsx

```diff
@@ -4,15 +4,15 @@ import BarcodeScan from './BarcodeScan';
 
 const ScanPatientPage = ({
   match: {
-    params: { ccpID },
+    params: { eventID, ccpID },
   },
 }: {
-  match: { params: { ccpID: string } };
+  match: { params: { eventID: string; ccpID: string } };
```

Comment:
out of curiosity, is there a particular reason these are type `string`? does changing them to `number` help us in any way (e.g. if you manually try to go to a URL where `eventId` is `123abc` - can we error out earlier)?

Reply:
I think it's because we had set the event ID to `string` in our GraphQL queries on the front end, just looking at the structs in `src/graphql/queries/events.ts`. I tried putting a mixture of numbers of alphabets and it failed  as it wasn't a valid ID (albeit the error message was slightly confusing to me). 
![Screen Shot 2020-08-26 at 8 08 04 AM](https://user-images.githubusercontent.com/21224282/91314009-4bf7ae80-e773-11ea-9fc8-b12e9c1f137a.png)

Reply:
hmm in that case no big deal, i'm keep an eye out for it if we do want to revisit this paradigm